package/chessmarkable/package

```diff
@@ -0,0 +1,27 @@
+# vim: set ft=sh:
+pkgname=chessmarkable
+pkgdesc="A chess game for the reMarkable"
+url=https://github.com/LinusCDE/chessmarkable
+pkgver=0.3.2-1
+timestamp=2020-10-04T22:12Z
+section=games
+maintainer="Linus K. <linus@cosmos-ink.net>"
+license=MIT
+
+image=rust:v1.1
+source=(https://github.com/LinusCDE/chessmarkable/archive/e97cb952cbbd806a94cea527ccd652fcf2302e50.zip)
+sha256sums=(4088c54d2db5de9fe56be99e3969dc5c1eb6a52d6462be67262839b84064cf22)
+
+build() {
+    # Fall back to system-wide config
+    rm .cargo/config
```

Comment:
I don't get it - did you need this to build in the docker image?

Reply:
I prefer to use the config to use the oecore toolchain and believe it's easier for users who want to compile it and don't have a cross compiler anyway. Mostly probably a habit of mine. I would like to keep it this way for a while though.

Reply:
if i understand right, this is falling back to the system wide config *inside* the docker image - that's what you intend, right?